site/content/posts/2021-06-22-understanding-kubernetes-tests.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Understanding Kubernetes E2E Tests
```

Comment:
Kubernetes tests in general, or windows e2e tests as the slug says?

Reply:
Slug is probably outdated; tests in general this time.